aspnet/SameSite/system-web-samesite.md

```diff
@@ -10,20 +10,59 @@ uid: samesite/system-web-samesite
 
-## API usage with SameSite
+Each ASP.NET component that emits cookies needs to decide if SameSite is appropriate.
 
-See [HttpCookie.SameSite Property](/dotnet/api/system.web.httpcookie.samesite#System_Web_HttpCookie_SameSite)
+Please see [Known Issues](#known) for problems with applications after installing the 2019 .Net SameSite updates.
+
+## Using SameSite in ASP.NET 4.7.2 and 4.8
+
+.Net currently supports the [2019 draft standard](https://tools.ietf.org/html/draft-west-cookie-incrementalism-00) for SameSite.
+Developers are able to programatically control the value of the SameSite header using the [HttpCookie.SameSite Property](/dotnet/api/system.web.httpcookie.samesite#System_Web_HttpCookie_SameSite). Setting this property to `Strict`, `Lax`, or `None`
+will result in those values being written on the network with the cookie. Setting it equal to `(SameSiteMode)(-1)` indicates that
+no SameSite header should be included on the network with the cookie. The [HttpCookie.Secure Property](/dotnet/api/system.web.httpcookie.secure) (or 'requireSSL' in config files) can be used to mark the cookie as `Secure` or not.
+
+New `HttpCookie` instances will default to `SameSite=(SameSiteMode)(-1)` and `Secure=false`. These defaults can be overridden in
+the `system.web/httpCookies` configuration section, where the string "Unspecified" is a friendly configuration-only syntax for `(SameSiteMode)(-1)`.
+```xml
+<configuration>
+ <system.web>
+  <httpCookies sameSite="[Strict|Lax|None|Unspecified]" requireSSL="[true|false]" />
+ <system.web>
+<configuration>
+```
+
+ASP.Net also issues four specific cookies of its own for these features: Anonymous Authentication, Forms Authentication, Session State,
+and Role Management. Instances of these cookies obtained in runtime can be manipulated using the `SameSite` and `Secure` properties
+just like any other HttpCookie instance. However, due to the patchwork emergence of the SameSite standard, configuration options for
+these four features cookies is inconsistent. The relevant configuration sections and attributes (with defaults) are below. If a
+SameSite/Secure-related attribute exists for a feature, then the feature will use it's own default or configured value. If there is
+no corresponding attribute, then the default will come from the `system.web/httpCookies` section.
+```xml
+<configuration>
+ <system.web>
+  <anonymousIdentification cookieRequireSSL="false" />
+  <authentication>
+   <forms cookieSameSite="Lax" requireSSL="false" />
+  </authentication>
+  <sessionState cookieSameSite="Lax" />
+  <roleManager cookieRequiresSSL="false" />
+ <system.web>
+<configuration>
+```  
+*Note: 'Unspecified' is only available to `system.web/httpCookies@sameSite` at the moment. We hope to add similar syntax to the cookieSameSite attributes above in future updates. Setting `(SameSiteMode)(-1)` in code still works on instances of these cookies.*
 
 ## History and changes
```

Comment:
> The November 19, 2019 updates for Windows updated .NET 4.7.2+ from the 2016 standard to the 2019 standard. Additional updates are forthcoming for other versions of Windows. For more information, see <xref:samesite/kbs-samesite>.

These additional patches have shipped.

Reply:
You've repeated almost this whole section in the opening section. Can we pair down one or the other?